draft-packages/button/KaizenDraft/Button/components/GenericButton.tsx

```diff
@@ -82,6 +89,17 @@ const GenericButton = forwardRef(
       },
     }))
 
+    if (
+      !props.iconButton &&
+      props.working !== undefined &&
+      !getWorkingLabel(props)
+    ) {
+      throw new Error(
```

Comment:
I definitely agree we need to warn the user somehow that they need to use the correct aria attributes when using `working`, but I am not sure that throwing an error is the way to go 🤔 If there's no error boundary (or something else to catch this error) then the page could crash in unexpected ways. I think `console.warn` or `console.error` might be more appropriate here. Thoughts?

Reply:
Yeah that's fair. I'll replace with console.error.

Reply:
Just want to throw one other idea in the mix - we might be able to enforce this with TypeScript.

If `working` is set to true, then you must specify a `workingLabel` prop. Then there can be an optional `workingLabelHidden` prop to decide whether it will be a visible label, or an `aria-label`